samples/DataSourceWithProcessingAndExtensions/Sample/MetadataTables/EventStatsTable.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using DataExtensionsSample.SourceDataCookers;
+
+namespace DataExtensionsSample.MetadataTables
+{
+    [Table]
+    [RequiresCooker(EventStatsCooker.CookerPathAsString)]
+    public class EventStatsTable
+    {
```

Comment:
Can we add a comment explaining how this Table gets its (cooked?) data? I'm assuming its through this RequiresCooker attribute. Maybe explain how these "CookerPathAsString" things work?